src/components/templates/AnimateMap/game/map/components/HoverableSpriteComponent.ts

```diff
@@ -1,3 +1,24 @@
 export class HoverableSpriteComponent {
-  constructor(public on?: Function, public off?: Function) {}
+  constructor(on?: Function, off?: Function, isCursorPointer: boolean = false) {
```

Comment:
Just by setting this newly added flag to default to `true`, changes in other files can be avoided, especially the one  
```ts
.add(new HoverableSpriteComponent(undefined, undefined, true));
```

```suggestion
  constructor(on?: Function, off?: Function, isCursorPointer: boolean = true) {
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/2161/commits/470154cca18bb5273e216b7d43acaace4ac25b4a